pkg/factory/risingwave_object_factory.go

```diff
@@ -397,7 +397,7 @@ func (f *RisingWaveObjectFactory) argsForMeta() []string {
 		"meta-node",
 		"--config-path", path.Join(risingwaveConfigMountPath, risingwaveConfigFileName),
 		"--listen-addr", fmt.Sprintf("0.0.0.0:%d", metaPorts.ServicePort),
-		"--host", "$(POD_IP)",
```

Comment:
can we use the `advertise_addr` option directly?

Reply:
No, we must keep it compatible for the current version. Please refer to the discussion

Reply:
Okay, that's fine.